erpnext/stock/doctype/stock_entry/stock_entry.py

```
@@ -138,6 +138,10 @@ class StockEntry(StockController):
 			"Repack",
 			"Send to Subcontractor",
 			"Disassemble",
+			"Receive from Customer",
```

Ruff (0.13.3)

Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)